device/accton/x86_64-accton_as5812_54t-r0/plugins/eeprom.py

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env python
```

Comment:
nit: shabang was removed from eeprom.py in other module, was this one missed?

Reply:
Yes, a miss, but harmless, as the file is not executable and shouldn't have had a shebang to begin with. There are other files which need the same treatment. I indent to make a pass later.